tests/baselines/reference/classExtendingClassLikeType.symbols

```diff
@@ -16,7 +16,6 @@ interface Base<T, U> {
 // Error, no Base constructor function
 class D0 extends Base<string, string> {
 >D0 : Symbol(D0, Decl(classExtendingClassLikeType.ts, 3, 1))
->Base : Symbol(Base, Decl(classExtendingClassLikeType.ts, 0, 0))

```

Comment:
I don’t think all these symbol baselines should have changed. I believe this means that `checker.getSymbolAtLocation` on these heritage clauses used to work, but now returns `undefined`.

Reply:
We looked at this in debugging; these are all cases that are in fact errors (this one is extending a class with a type-only thing like an Interface) and don't even get quickinfo before this change, so I think this is actually okay.

Reply:
Ah ok, that makes sense.

Reply:
Oh, and just to note it, we tried renaming it and that also didn't work; these both turn out to behave the same as if you didn't have an `ExpressionWithTypeArguments`, like:

```ts
interface Base {
    prop: number
}

class D0 extends Base { 
}
```